obspy/clients/fdsn/__init__.py

```diff
@@ -134,8 +134,69 @@
                                         endtime=endtime)
         inventory.plot()
 
+
+Basic FDSN FedCatalog Client Usage
+----------------------------------
+
+The
+:mod:`FDSN fedcatalog_client <obspy.clients.fdsn.routers.fedcatalog_client>`
+module provides federated
+access to multiple web servers that implement the
+`FDSN Station and Dataselect web service definitions
+<https://www.fdsn.org/webservices/>`_.
+
+The first step is always to initialize a :class:`FederatedClient` object.
+
+>>> from obspy.clients.fdsn import FederatedClient
+>>> client = FederatedClient()
+
+(1) :meth:`~obspy.clients.fdsn.routers.fedcatalog_client.
+FederatedClient.get_waveforms()`: The following
```

Comment:
broken link here too, likely due to aforementioned missing submodule entries in sphinx skeleton document (`misc/docs/source/packages/obspy.clients.fdsn.rst`)